server/views/pages/barcode/find-recipient.njk

```diff
@@ -32,7 +32,7 @@
           <p>Barcode number: {{ barcode }}</p>
           <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
-              <img alt="The generated barcode image" src="{{ barcodeImageUrl }}" id="barcodeImage" style="display: block">
+              <img alt="The generated barcode image" src="{{ barcodeImageUrl }}" id="barcodeImage">
```

Comment:
The use of inline styling causes chrome (and probably other browsers) to moan - it still works, but you get a console error.
Personally I think this element looks better if not display:block (ie. default of display:inline), but if we really want to make it block then we should use the GDS class `govuk-!-display-block`
eg:
```
<img alt="The generated barcode image" src="{{ barcodeImageUrl }}" id="barcodeImage" class="govuk-!-display-block">
```

Reply:
Yeah I spotted that after my PR was already in - but it's not staying on that page anyway so all good.